.github/workflows/check_consistency.yml

```
@@ -18,11 +18,31 @@ jobs:
       PYTHONPATH: ${{ github.workspace }}
 
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v5
         with:
           fetch-depth: 0
+          persist-credentials: false
 
       - uses: ./.github/steps/prepare_minimal_ubuntu_image
```

actionlint (1.7.12)

[error] description is required in metadata of "Prepare FORGE minimal ubuntu image" action at "/home/jailuser/git/.github/steps/prepare_minimal_ubuntu_image/action.yml"

(action)